packages/sanity/src/core/studio/components/navbar/appearance/appearance.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import {Box, Button, Label, Stack, useRootTheme} from '@sanity/ui'
+import {CheckmarkIcon, DesktopIcon, MoonIcon, SunIcon} from '@sanity/icons'
+import styled from 'styled-components'
+import {useColorScheme} from '../../../colorScheme'
+import {StudioTheme} from '../../../../theme'
+
+const StyledButton = styled(Button)`
+  cursor: pointer;
```

Comment:
The current convention in the studio is that only buttons with URLs should have a hand cursor. Given that these are commands, we can skip creating a custom style and remove this altogether.